content/en/blog/news/2025-03-kcp-at-kubecon-london.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Our Schedule at KubeCon + CloudNativeCon Europe 2025"
+linkTitle: "Our Schedule at KubeCon + CloudNativeCon Europe 2025"
+slug: kcp-at-kubecon-cloudnativecon-eu-2025
+date: 2025-03-27
+tags:
+  - cncf
+  - community
+  - kubecon
+  - cloudnativecon
+  - talks
+  - conference
+authors:
+  - Marko Mudrinić
+---
+
+[KubeCon + CloudNativeCon Europe 2025](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/) is just a **few days away**! The major European cloud native conference is set to kick off on Tuesday, April 1 in London, England!
+
+The kcp project has prepared a lot of exiciting content and activites for this year's KubeCon EU. Just like at KubeCon NA last year, you'll have a chance to meet our maintainers at the **Project Pavilion** on **Wednesday, April 2, from 15:30 to 19:45 BST**. Feel free to drop by and talk anything about kcp or just say hi. 👋
```

Comment:
Let's add the kiosk no, which is 5A (see https://events.linuxfoundation.org/wp-content/uploads/2025/03/KubeConEU25_SSMap_v1.pdf).

Reply:
Addressed, let me know if it's okay.